modules/tensor_mechanics/src/materials/RadialReturnStressUpdate.C

```diff
@@ -196,41 +196,51 @@ RadialReturnStressUpdateTempl<false>::updateState(RankTwoTensor & strain_increme
   // compute the effective trial stress
   Real dev_trial_stress_squared =
       deviatoric_trial_stress.doubleContraction(deviatoric_trial_stress);
-  Real effective_trial_stress = std::sqrt(3.0 / 2.0 * dev_trial_stress_squared);
 
-  // Set the value of 3 * shear modulus for use as a reference residual value
-  _three_shear_modulus = 3.0 * ElasticityTensorTools::getIsotropicShearModulus(elasticity_tensor);
-
-  computeStressInitialize(effective_trial_stress, elasticity_tensor);
-
-  // Use Newton iteration to determine the scalar effective inelastic strain increment
-  _scalar_effective_inelastic_strain = 0.0;
-  if (!MooseUtils::absoluteFuzzyEqual(effective_trial_stress, 0.0))
+  if (MooseUtils::absoluteFuzzyEqual(dev_trial_stress_squared, 0.0))
```

Comment:
There's not a way to comment on a line of code you haven't changed, but my only comment on this PR is that with your changes, in line 149 of this file, that check on `_scalar_effective_inelastic_strain` being equal to zero should always be false. Because of that, I think we should get rid of the if/else check starting on 149 and change that to be an assert like line 160.

Reply:
I just noticed that `computeTangentOperator` is also called from `updateStateSubstep`, where this requirement might not be satisfied.